Makefile

```diff
@@ -1,7 +1,4 @@
-faker-schema-refresh:
-	docker-compose stop graphql-faker && docker rm graphql-faker && docker image rm graphql-faker && docker-compose up -d --build graphql-faker
-dev-staging:
+dev:
 	docker compose up geyser-app caddy
-dev-faker:
-	docker compose up
-	
\ No newline at end of file
+test:
```

Comment:
Maybe we can rename this to "test-e2e"?

Reply:
yeah, once we have other commands, it would make sense, it's okay for now.